server/auto_updater/get_square_bins.R

```diff
@@ -0,0 +1,9 @@
+library(ggplot2)
```

Comment:
Can you tell me what this script is for? I didn't find it invoked anywhere.

Reply:
This script is used to generate the square bin plot that you see on the main page of the web app. There aren't any python packages that can generate this plot while allowing the user to access each individual square bin. Hence the need to use R in this situation.